app/concepts/list/operation/delete.rb

```diff
@@ -3,10 +3,15 @@
 module List::Operation
   class Delete < Trailblazer::Operation
     step Model(List, :find_by)
+    step :set_result
     step :destroy_list
```

Comment:
I would swap these steps. The result will be assigned to the context even if the destroy fails. I think it's incorrect